### ISSUE TITLE
Enhancement of "Contact Us!" form

### ISSUE THREAD

**Issue by AditiPrabhakar** [khushi-joshi-05/Food-ordering-website#446]
Contact Us! form looks very basic and the designing is not good, i can improve it, also it would be better to include a name field to take users name input

Please assign this to me under gssoc24 

![Screenshot (96)](https://github.com/khushi-joshi-05/Food-ordering-website/assets/139689410/2580615d-4032-408a-8ae5-905a0304674d)

**Comment by SoumyajitPr** [comment]
I agree with this issue, but I can design and even, add some more features in the field. Please assign me this issue under GSSOC '24, and I won't disappoint. Thank you.

**Comment by Hanshika248** [comment]
Please Assign me the issue

**Comment by KomalSrivastava** [comment]
@khushi-joshi-05 Let's enhance the "Contact Us!" form subtly yet significantly. Introducing a name field will add a personal touch to the user experience. With my dedication and eye for detail, I'll ensure that these enhancements align perfectly with your vision for the website. Adding a name field will personalize the interaction and improve user engagement.

Please assign this task to me for GSSOC24, and let's make the user interaction more engaging.
![contactus](https://github.com/khushi-joshi-05/Food-ordering-website/assets/132221203/91ef12ab-64e3-4e48-bd6a-f943a36debcd)
